pages/maps/index.vue

```diff
@@ -30,7 +31,87 @@
 
 <script>
 import Breadcrumb from '@/components/Breadcrumb/Breadcrumb.vue'
+import FetchPennsieveFile from '@/mixins/fetch-pennsieve-file'
+import flatmaps from '@/services/flatmaps'
 import PageHero from '@/components/PageHero/PageHero.vue'
+import scicrunch from '@/services/scicrunch'
+
+import { failMessage } from '@/utils/notification-messages'
+
+const getFlatmapEntry = async ( route ) => {
+  const uberonid = route.query.uberonid
+  let organ_name = undefined
+  try {
+    organ_name = await scicrunch.getOrganFromUberonId(uberonid)
```

Comment:
I think something in this is broken, I'll check it out

Reply:
Is it from the Scaffold dataset? Scaffold datasets do not have the correct information.

Reply:
yeah, I think it is just the way I made that function. I'm looking into it now